src/common/exception/ErrorMessage.jl

```diff
@@ -0,0 +1,220 @@
+# This file is a part of GraknClient.  License is MIT: https://github.com/Humans-of-Julia/GraknClient.jl/blob/main/LICENSE 
+
+# Devide the ErrorClassification Client, Concept, Query and Internal is possible
+# with the following abstrat types. 
+abstract type AbstractGeneralError end
+abstract type AbstractClientError <: AbstractGeneralError end
+abstract type AbstractConceptError <: AbstractGeneralError end
+abstract type AbstractQueryError <: AbstractGeneralError end
+abstract type AbstractInternalError <: AbstractGeneralError end
+
+# The Symbol left side is assigned to the struct which will be build bellow
+# The right side of the Pair is assigned to the abstract type representing the section of error
+const error_structs = (
+    :Client_CLIENT_CLOSED=>:AbstractClientError,
```

Comment:
Sample style suggestions. Use UPPER_CASE for constants and put spaces around operators.

```suggestion
const ERROR_STRUCTS = (
    :CLIENT_CLIENT_CLOSED => :AbstractClientError,
```